### ISSUE TITLE
Focus not always visible on JupyterLab initial page

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/jupyter-a11y-mgmt#167]
### Description

On the initial page in JupyterLab, some elements that receive focus by pressing the tab key do not have a visible focus indicator, which is a violation of WCAG 2.4.7.

This issue is about fixing those focus indicators for the initial page in JupyterLab.

### Value / benefit

[Understanding WCAG 2.4.7](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html) explains why.

```[tasklist]
### TODO
- [ ] https://github.com/Quansight-Labs/jupyter-a11y-mgmt/issues/199
- [x] Submit PR(s) that make the test pass
- [ ] https://github.com/Quansight-Labs/jupyter-a11y-testing/pull/33
```

**Comment by gabalafou** [comment]
https://github.com/jupyterlab/jupyterlab/pull/13415

**Comment by gabalafou** [comment]
Update on this issue. 

This issue stalled on the test-writing sub-task because I couldn't make the test that I wrote pass. I believe that [Mark's PR](https://github.com/jupyterlab/lumino/pull/465) fixed the issue, but I need to go back and try testing it.

**Comment by trallard** [comment]
@gabalafou can you remind me what we said about this particular issue, will you be working on it?

**Comment by gabalafou** [comment]
This issue has 2 parts:

1. fix 
2. regression test

The first part is done. The second part is still outstanding, and I plan to work on it in the upcoming weeks as part of my focus on accessibility testing in order to both integrate accessibility regression testing with general JupyterLab testing; and to get ready for my talk at JupyterCon.

**Comment by gabalafou** [comment]
By the way, this is the PR for the fix:

- https://github.com/jupyterlab/jupyterlab/pull/13415

**Comment by trallard** [comment]
Regression tests added - need more things fixed

**Comment by gabalafou** [comment]
I'm going to mark this as closed, even though there are some things that are still in progress:

- [Focus visible (initial page) regression test](https://github.com/Quansight-Labs/jupyter-a11y-testing/pull/33) is in review, it has not been merged yet
- Regression test fails because the skip link is broken (not visible), but there is a PR open to [fix the skip link](https://github.com/jupyterlab/jupyterlab/pull/14597)

I'm going to mark it as closed because the bulk of the work that I can do for this issue is already done.
